Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/Page.cs

```diff
@@ -56,5 +56,82 @@ public static UIStatusBarAnimation PreferredStatusBarUpdateAnimation(this IPlatf
 			SetPreferredStatusBarUpdateAnimation(config.Element, value);
 			return config;
 		}
+
+		public static readonly BindableProperty UseSafeAreaProperty = BindableProperty.Create(nameof(UseSafeArea), typeof(bool), typeof(Page), false);
+
+		public static bool GetUseSafeArea(BindableObject element)
+		{
+			return (bool)element.GetValue(UseSafeAreaProperty);
+		}
+
+		public static void SetUseSafeArea(BindableObject element, bool value)
+		{
+			element.SetValue(UseSafeAreaProperty, value);
+		}
+
+		public static IPlatformElementConfiguration<iOS, FormsElement> SetUseSafeArea(this IPlatformElementConfiguration<iOS, FormsElement> config, bool value)
+		{
+			SetUseSafeArea(config.Element, value);
+			return config;
+		}
+
+		public static bool UseSafeArea(this IPlatformElementConfiguration<iOS, FormsElement> config)
+		{
+			return GetUseSafeArea(config.Element);
+		}
+
+		public static readonly BindableProperty LargeTitleDisplayProperty = BindableProperty.Create(nameof(LargeTitleDisplay), typeof(LargeTitleDisplayMode), typeof(Page), LargeTitleDisplayMode.Automatic);
+
+		public static LargeTitleDisplayMode GetLargeTitleDisplay(BindableObject element)
+		{
+			return (LargeTitleDisplayMode)element.GetValue(LargeTitleDisplayProperty);
+		}
+
+		public static void SetLargeTitleDisplay(BindableObject element, LargeTitleDisplayMode value)
+		{
+			element.SetValue(LargeTitleDisplayProperty, value);
+		}
+
+		public static LargeTitleDisplayMode LargeTitleDisplay(this IPlatformElementConfiguration<iOS, FormsElement> config)
+		{
+			return GetLargeTitleDisplay(config.Element);
+		}
+
+		public static IPlatformElementConfiguration<iOS, FormsElement> SetLargeTitleDisplay(this IPlatformElementConfiguration<iOS, FormsElement> config, LargeTitleDisplayMode value)
+		{
+			SetLargeTitleDisplay(config.Element, value);
+			return config;
+		}
+
+		public static readonly BindableProperty SafeAreaInsetsProperty = BindableProperty.Create(nameof(SafeAreaInsets), typeof(Thickness), typeof(Page), default(Thickness), propertyChanged: (bindable, oldValue, newValue) =>
```

Comment:
oh, this is what I was looking for...

we probably do not need both.

is this inset also used for cell insets ?